addons/website_profile/static/src/scss/website_profile.scss

```diff
@@ -125,6 +125,21 @@ $owprofile-color-bg: mix($body-bg, #efeff4);
 
     .o_pc_overlay {
         @include o-position-absolute(0,0,0,0);
+        // Allow hover effect on svg paths in order to display bootstrap tooltip,
+        // by pushing the absolute div behind it
+        z-index: -1;
```

Comment:
Hum seems like this line broke the mobile view (the rank image disappear), can you check if you can do something ?
![Screenshot from 2022-08-11 17-20-17](https://user-images.githubusercontent.com/49644838/184169219-be87f2e8-7991-4e75-bfcb-44dd83fae833.png)